### ISSUE TITLE
Dropdown: when switch option from a dropdown, screen reader for selected option will be skipped if there's other component's alert read out 

### ISSUE THREAD

**Issue by gggjtgxing** [OfficeDev/office-ui-fabric-react#780]
### Bug Report

screen reader: JAWS Version 17.0.2619
browser: Internet Explorer 11

#### Describe the issue:

After choosing an option from dropdown, it read out the option text immediately, but if at the same time there's another alert read out something, dropdown screen reader will be skipped.

#### Actual behavior:
A scenario for this issue:
I use key down to select 'option d', another control read out 'alert, your page has been saved', and reading nothing for dropdown.

#### Expected behavior:
After one option is selected, screen reader should read 'option d' first, then start reading other alert .

**Comment by cschlechty** [comment]
The text should be queued in most screen readers. Which browser and screen reader?

**Comment by gggjtgxing** [comment]
I filled 'bug report' field with browser and screen reader version.
Here is a scenario:
I use key down to select 'option d', another control read out 'alert, your page has been saved', and reading nothing for dropdown. I expect  this: read 'option d' first, then read other alert 'alert, your page .....';
